src/main/resources/views/CommercialVehicles/failWelsh.hbs

```diff
@@ -33,7 +33,7 @@
         </div>
         <div class="footer__certificate-tag">
                <span class="footer__certificate-tag-text">
-               {{presentedDocumentNameFail}}/{{versionNumberFail}}
+               {{presentedDocumentNameFail}}/{{VersionNumberFailWelsh}}
```

Comment:
Sorry, I may not have been clear with the last comment, this needs to display the Welsh for both.

So VTP30W/1.0

It would be the same for line 73.